roles/monitored/tasks/raid.yml

```diff
@@ -20,13 +20,15 @@
   # download the zip from broadcom
   # extract the .rpm
   # extract the binary out of it and drop it in /usr/local
+  # patch it to work with modern ncurses
   shell: > # noqa: command-instead-of-module
     set -o pipefail &&
     wget -q -O - https://docs.broadcom.com/docs-and-downloads/raid-controllers/raid-controllers-common-files/8-07-14_MegaCLI.zip
     | busybox unzip -p - Linux/MegaCli-8.07.14-1.noarch.rpm
     | rpm2cpio -
     | cpio --quiet -i --to-stdout ./opt/MegaRAID/MegaCli/MegaCli64 > /usr/local/bin/megacli
     && chmod +x /usr/local/bin/megacli
+    && patchelf --replace-needed libncurses.so.5 libncurses.so.6 /usr/local/bin/megacli
```

Comment:
We should  try and only run this when the source file has changed.

Can I suggest downloading the zip in a separate task and then setting a var to trigger re-processing it when_changed?

Reply:
Also any command  we can use to test that megacli still works afterwards?

Reply:
the whole thing has a `creates` for the binary - so it'll only ever run once per host

that also means it won't touch the existing binaries on our older hosts, which may be problematic as we upgrade them - but we can always just fix those by manually deleting the binary and re-running ansible